### ISSUE TITLE
A11y_ML .NET Model Builder_ Advance Data Options_ Screen Reader: Narrator/NVDA are not reading the label "Does the data contain column headers?"

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1120]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev2; #NonBenchmark; #Visual Studio; #Screen Reader; #Narrator; #NVDA; #WCAG1.3.1;

### Environment Details:
Microsoft Visual Studio 2019 Preview Version 16.8.0 Preview 6.0

### Screen Readers:
Narrator; NVDA

### Repro Steps:

1.Start Screen Reader
2.Open Visual Studio Enterprise 2019 Preview 
3. Create a new .Net Core Console App (C#) 
4. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
5. Tab till "Text Classification" scenario and select the scenario
6. Tab till "Next Step" button and press enter
7. Select the data source type and upload the file
8. Tab till "Advanced Data Options" link and press enter
9. Tab till "Data Formatting" tab item and press enter
10. Verify whether Screen readers are reading the label "Does the data contain column headers?" or not

### Actual Result:
Narrator/NVDA are not reading the label "Does the data contain column headers?". When focus comes to "Yes" radio button screen readers are reading as "Yes radio button selected"

### Expected Result:
When focus comes to first radio button screen readers should first read the label of the radio buttons group and then the radio button name. Screen readers should read as "Does the data contain column headers?  Yes radio button selected"       

### User Impact:
Screen Reader users will not get to know what are these radio buttons about if screen reader doesn't read the label

### Attachment for Reference

[1120_ML .NET Model Builder_ Narrator is not reading the label for radio buttons.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5607556/1120_ML.NET.Model.Builder_.Narrator.is.not.reading.the.label.for.radio.buttons.pptx)
[1120_ML .NET Model Builder_ NVDA is not reading the label for radio buttons.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5607686/1120_ML.NET.Model.Builder_.NVDA.is.not.reading.the.label.for.radio.buttons.pptx)

**Comment by SaisreeDandamudi** [comment]
Verified the bug in the given build and the issue is fixed. Hence closing the bug
Please find the below screen shots for reference
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.9.0 Preview 4.0 [30917.4.main]
![Screenshot (54)](https://user-images.githubusercontent.com/51435580/108032326-50cd5900-7058-11eb-96f8-7349cf37e6e6.png)
![Screenshot (55)](https://user-images.githubusercontent.com/51435580/108032333-53c84980-7058-11eb-98a8-80c868fa644d.png)
